extensions/2.0/Khronos/KHR_materials_ior/schema/glTF.KHR_materials_ior.schema.json

```
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "KHR_materials_ior glTF extension",
+    "type": "object",
+    "description": "glTF extension that defines the index of refraction of a material.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "ior": {
+            "type": "number",
+            "description": "The index of refraction.",
+            "default": 1.5,
+            "minimum": 1.0,
```

We talked offline today about allowing zero as a special-case value.  This snippet of schema would implement that allowance.

```suggestion
            "oneOf": [
                {
                    "minimum": 0.0,
                    "maximum": 0.0
                },
                {
                    "minimum": 1.0
                }
            ],
```

---

@emackey Thank you! I updated the ior and specular extensions.